eng/common-variables.yml

```diff
@@ -0,0 +1,34 @@
+variables:
+  # Cannot use key:value syntax in root defined variables
+  - name: _TeamName
+    value: DotNetCore
+  - name: HelixApiAccessToken
+    value: ''
+  - name: _RunAsPublic
+    value: True
+  - name: _RunAsInternal
+    value: False
+  - name: _InternalBuildArgs
+    value: ''
+
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - name: _RunAsPublic
+      value: False
+    - name: _RunAsInternal
+      value: True
+    - name: _SignType
+      value: real
+    # DotNet-Blob-Feed provides: dotnetfeed-storage-access-key-1
+    # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
+    # DotNet-HelixApi-Access provides: HelixApiAccessToken
+    - group: DotNet-Blob-Feed
+    - group: Publish-Build-Assets
+    - group: DotNet-HelixApi-Access
+    - group: SDL_Settings
```

Comment:
This is too much of a direct copy of Arcade's values, you don't need all these variable groups and SDL_Settings is specific to Arcade-y repos.   
For CodeQL purposes you just need variables for these values:
``` 
_TsaBugAreaPath 
_TsaCodebaseAdmin
_TsaInstanceURL
_TsaIterationPath
_TsaNotificationEmail
_TsaProjectName
_TsaTemplateName
```

Reply:
Updated the public variables.